milv.config.yaml

```diff
@@ -1,6 +1,26 @@
-white-list-external: ["localhost", "console.kyma.local"]
+white-list-external: ["localhost", "kyma.local", "CLUSTER_NAME.kyma.cluster.cx", "$yourClusterDomain", "svc.cluster.local:8081"]
 black-list: ["/vendor/"]
 files:
+  - path: "./kyma/CONTRIBUTING.md"
+    links:
+      - path: "https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md"
+        config:
+          request-repeats: 5
```

Comment:
`request-repeats` should be defined on global level as every `external` link may fail due to connection problem.